css/src/components/gradient-card.scss

```diff
@@ -0,0 +1,11 @@
+.gradient-card {
+	border: $border-width-md solid $border !important;
+	border-radius: $border-radius-lg !important;
+	border-color: transparent !important;
+	background: linear-gradient(90deg, $gradient-vivid-start, $gradient-vivid-end) border-box !important;
+
+	.gradient-card-content {
+		border-radius: $border-radius !important;
```

Comment:
Applying `overflow: hidden` on the `gradient-card` level can eliminate `border-radius` here

Reply:
Great tip, worked well. Updated.